openshift_tooling/kraken/kraken.py

```diff
@@ -227,7 +276,12 @@ def master_test(label, master_label):
     master_node = get_random_node(master_label)
     print (Fore.GREEN + 'killing %s\n') %(master_node)
     #cmd = "systemctl stop atomic-openshift-master-controllers.service"
-    cmd = "systemctl stop atomic-openshift-master-apiserver.service"
+    ##cmd = "systemctl stop atomic-openshift-master-apiserver.service"
+    # For OCP 3.10: we have static pods:
+    # This is for restart:
+    # cmd = "/usr/local/lib/master-restart api"
```

Comment:
I think it should be master-restart api api  and master-restart controllers controllers

In any case, please remove obsolete comments here and for etcd restart.   Otherwise lgtm